webapp-django/crashstats/crashstats/tests/test_models.py

```diff
@@ -198,110 +198,103 @@ def new_mocked_get(**options):
 
-    @mock.patch('requests.get')
-    def test_current_products(self, rget):
+    def test_current_products(self):
         api = models.CurrentProducts()
 
-        def mocked_get(url, params, **options):
-            assert '/products' in url
-
-            if 'versions' in params and params['versions'] == 'WaterWolf:2.1':
-                return Response("""
-                {
-                  "hits": [
-                    {
-                        "is_featured": true,
-                        "throttle": 1.0,
-                        "end_date": "string",
-                        "start_date": "integer",
-                        "build_type": "string",
-                        "product": "WaterWolf",
-                        "version": "15.0.1",
-                        "has_builds": true
-                    }],
-                    "total": "1"
+        def mocked_get(**options):
+            if options.get('versions') == 'WaterWolf:2.1':
+                return {
+                    'hits': [
+                        {
+                            'is_featured': True,
+                            'throttle': 100.0,
+                            'end_date': 'string',
+                            'start_date': 'integer',
+                            'build_type': 'string',
+                            'product': 'WaterWolf',
+                            'version': '15.0.1',
+                            'has_builds': True
+                        }
+                    ],
+                    'total': 1
                 }
-                """)
 
             else:
-                return Response("""
-                {
-                  "hits": [
-                    {
-                        "sort": "1",
-                        "default_version": "15.0.1",
-                        "release_name": "firefox",
-                        "rapid_release_version": "5.0",
-                        "product_name": "NightTrain"
-                    }],
-                    "total": "1"
+                return {
+                    'hits': [
+                        {
+                            'is_featured': True,
+                            'throttle': 100.0,
+                            'end_date': 'string',
+                            'start_date': 'integer',
+                            'build_type': 'string',
+                            'product': 'NightTrain',
+                            'version': '15.0.1',
+                            'has_builds': False
+                        }
+                    ],
+                    'total': 1
                 }
-                """)
 
-            raise NotImplementedError(url)
+        models.ProductVersions.implementation().get.side_effect = mocked_get
 
-        rget.side_effect = mocked_get
         info = api.get()
-        eq_(info['hits'][0]['product_name'], 'NightTrain')
+        eq_(info['hits']['NightTrain'][0]['product'], 'NightTrain')
```

Comment:
Was this not supposed to stay the same? As far as I understand if you had to change this test that means the output of ``CurrentProducts`` changed, right?